demos/jobs/jobs_demo_mosquitto/CMakeLists.txt

```diff
@@ -53,32 +53,8 @@ target_include_directories(
         ${JSON_INCLUDE_PUBLIC_DIRS}
 )
 
-if(AWS_IOT_ENDPOINT)
-    target_compile_definitions(
-        ${DEMO_NAME} PRIVATE
-            AWS_IOT_ENDPOINT="${AWS_IOT_ENDPOINT}"
-    )
-endif()
-if(ROOT_CA_CERT_PATH)
-    target_compile_definitions(
-        ${DEMO_NAME} PRIVATE
-            ROOT_CA_CERT_PATH="${ROOT_CA_CERT_PATH}"
-    )
-endif()
-if(CLIENT_CERT_PATH)
-    target_compile_definitions(
-        ${DEMO_NAME} PRIVATE
-            CLIENT_CERT_PATH="${CLIENT_CERT_PATH}"
-    )
-endif()
-if(CLIENT_PRIVATE_KEY_PATH)
-    target_compile_definitions(
-        ${DEMO_NAME} PRIVATE
-            CLIENT_PRIVATE_KEY_PATH="${CLIENT_PRIVATE_KEY_PATH}"
-    )
-endif()
 set_macro_definitions(TARGETS ${DEMO_NAME}
```

Comment:
The jobs demo does not expose connection setting macros, so the `set_macro_definitions` call can be removed

Reply:
I believe it can be passed through here
https://github.com/aws/aws-iot-device-sdk-embedded-C/blob/main/demos/jobs/jobs_demo_mosquitto/jobs_demo_mosquitto.c#L430-L446

However, I set these parameters as optional as they can be passed at runtime.

Reply:
Interesting